lib/bots/versionbot.ts

```diff
@@ -680,69 +701,62 @@ export class VersionBot extends ProcBot {
 		const author = prEvent.sender.login;
 		let committer = author;
 		let lastCommit: GithubApiTypes.Commit;
+		let botConfig: VersionBotConfiguration;
 
 		// Only for opened or synced actions.
 		if ((event.cookedEvent.data.action !== 'opened') && (event.cookedEvent.data.action !== 'synchronize') &&
 			(event.cookedEvent.data.action !== 'labeled')) {
 			return Promise.resolve();
 		}
 
-		this.logger.log(LogLevel.INFO, `Checking version for ${owner}/${name}#${pr.number}`);
-		return this.dispatchToEmitter(this.githubEmitterName, {
-			data: {
+		this.logger.log(LogLevel.INFO, `Checking footer tags for ${owner}/${name}#${pr.number}`);
+
+		// Get the configuration, if it exists, for this repo.
+
+		// Get all the commits on the repo.
+
+		// For each tag we find, we adhere to some rules:
+		//  * all - Every commit in the PR must contain the tag
+		//  * once - At least one commit in the PR must contain the tag
+		//  * <integer> - There must be at least <integer> commits with the tag. If there
+		//                are less commits than the given integer, then all commits must
+		//                include the tag.
+		// Get the reviews for the PR.
```

Comment:
These various comments here seem a bit out of place. Should each of them be next to the sections below that are actually doing what they're describing?